src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_base.cpp

```diff
@@ -111,10 +115,24 @@ KernelsData FullyConnectedKernelBase::GetCommonKernelsData(const Params &params,
                      true,
                      !orgParams.bias.empty(),
                      1,
-                     fused_deps_total,
+                     0,
                      1,
                      orgParams.outputs[0].is_dynamic());
 
+    if (newParams.compressed) {
+        kernel.params.arguments.push_back({ArgumentDescriptor::Types::INPUT, 1});
+        if (newParams.has_decompression_zp)
+            kernel.params.arguments.push_back({ArgumentDescriptor::Types::INPUT, 2});
+    }
+
+    uint32_t fused_deps_total = 0;
+    for (const auto& fused_dep : newParams.fused_ops) {
+        for (int i = 0; i < static_cast<int>(fused_dep.dep_size); i++) {
+            kernel.params.arguments.push_back({ ArgumentDescriptor::Types::INPUT_OF_FUSED_PRIMITIVE, fused_deps_total });
+            fused_deps_total++;
+        }
+    }
```

Comment:
I think we can use `FillCLKernelData`'s `number_of_inputs` parameter instead of manual handling of inputs and fused_inputs args

Reply:
Updated